app/Http/Controllers/JobController.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Job;
+use App\Tag;
+
+class JobController extends Controller
+{
+
+    public function index()
+    {
+        $skill = Tag::where('type', 1)->get();
+        $lang = Tag::where('type', 2)->get();
+        $workPermit = Tag::where('type', 3)->get();
+
+        return view('listjob', [
+            'jobs' => Job::orderBy('created_at', 'asc')->get(),
+            'skill' => $skill,
+            'lang' => $lang,
+            'workPermit' => $workPermit,
+        ]);
+    }
+
+    public function create()
+    {
+        $skill = Tag::where('type', 1)->get();
+        $lang = Tag::where('type', 2)->get();
+        $workPermit = Tag::where('type', 3)->get();
+        $company_id = Auth::user()->company()->id;
+
+        return view('create_job', [
+            'skill' => $skill,
+            'lang' => $lang,
+            'workPermit' => $workPermit,
+            'id' =>$company_id,
+        ]);
+    }
+
+    public function store(Request $request)
+    {
+        $job = $request->all();
+        Job::create($job);
+        $job = Job::orderBy('created_at', 'desc')->first();
```

Comment:
em lấy job vừa được tạo à?
có thể dùng 
```suggestion
        $job = Job::create($job);
```